File: 1304. Find N Unique Integers Sum up to Zero/solution.cs
public class Solution {
    public int[] SumZero(int n) {
        var res = new int[n];
        var sum = 0;

        for (var idx = 0; idx < n - 1; idx++) {
            var current = idx + 1;
            res[idx] = current;
            sum += current;
        }

        res[n-1] = -sum;
        return res;
    }
}

File: 1304. Find N Unique Integers Sum up to Zero/solution.php
<?php

class Solution
{

    /**
     * @param Integer $n
     * @return Integer[]
     */
    function sumZero($n)
    {
        $res = [];
        $sum = 0;

        for ($idx = 0; $idx < $n - 1; $idx++) {
            $current = $idx + 1;
            $res[] = $current;
            $sum += $current;
        }

        $res[] = -$sum;
        return $res;
    }
}
